clients/drcachesim/tests/skip_unit_tests.cpp

```diff
@@ -154,12 +164,57 @@ test_skip_initial()
         std::istringstream ref_in(line);
         ref_in >> ref_count;
         CHECK(ref_count > skip_instrs, "invalid ref count");
-        // Reset cerr.
-        std::cerr.rdbuf(prior);
     }
     return true;
 }
 
+bool
+test_skip_indirect_marker()
+{
+    // Our trace has an indirect branch at this point:
+    static constexpr int INDIRECT_ORD = 132;
+    int skip_instrs = INDIRECT_ORD - 1;
+    std::string res;
+    if (!run_view_tool(skip_instrs, res))
+        return false;
+    std::stringstream res_stream(res);
+    // What we expect:
+    //   Output format:
+    //   <--record#-> <--instr#->: <---tid---> <record details>
+    //   ------------------------------------------------------------
+    //            202         131:     1260485 <marker: timestamp 13335506424391481>
+    //            203         131:     1260485 <marker: tid 1260485 on core 1>
```

Comment:
So without this PR, we would get 205 onwards? And now this PR makes sure we get 204 also, and 202-203 are due to existing logic of duplicating timestamp-tid? Maybe add a comment about why this is the expected output.